core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -746,7 +746,7 @@ class ContainerPool(
 
     // Getter/setter for this are above.
     private var _logDir = "/logs"
-    private val actionContainerPrefix = "wsk"
+    private val actionContainerPrefix = s"wsk${invokerInstance.toInt}_"
```

Comment:
Does the `_` change the container naming? Will there be `__` (two underscores) afterwards?